forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -386,8 +443,13 @@ useEffect(() => {
           text={t("Loading...")}
         >
 
-<SettingsModal show={showSettingsModal} handleClose={handleCloseModal} 
-handleConfirm={handleConfirmSettings} />
+          <SettingsModal show={showSettingsModal} handleClose={handleCloseModal}
+            handleConfirm={handleConfirmSettings} 
+            rolesState={rolesState} setRolesState={setRolesState} 
+            setFormDetails={setFormDetails} formDetails={formDetails} 
+            updateFormName={updateFormName} formType={formType} setFormType={setFormType} 
+            updateFormDescription={updateFormDescription} newPath={newPath} 
+            handleFormPathChange={handleFormPathChange}/>
```

Comment:
Can we handle the functionalities like these from settingsmodal itself instead of Edit.js Only the callbackfunction after Save Changes button click from Settings modal need to be handled in edit.js @Bonymol-aot

Reply:
Will update